sbom.json

```diff
@@ -0,0 +1,7 @@
+{
+  "serialNumber": "urn:uuid:a291eaa6-9c96-4c46-9fb1-474f745cf6f5",
```

Comment:
`a291eaa6-9c96-4c46-9fb1-474f745cf6f5` is just a random UUID, which is fine according to Ryan Timmons.